assets/js/admin/marketplace-suggestions.js

```diff
@@ -49,20 +49,24 @@
 			}
 
 			window.wcTracks.recordEvent( 'marketplace_suggestion_dismissed', {
-				suggestion_slug: suggestionSlug
+				suggestion_slug: suggestionSlug,
+				context: context,
+				product: product,
```

Comment:
If for some reason `product`, `promoted`, or `url` are undefined/null do we still want to record the track?

Reply:
Yes we do! These are essentially optional attributes that are supplied from the json end. We want to be robust to them missing for whatever reason.

Even just the fact that the event occurred (e.g. `suggestion_dismissed`) is useful data.